os/Kconfig

```diff
@@ -198,6 +198,42 @@ config BUILD_KERNEL
 
 endchoice # Build configuration
 
+config FLASH_START_ADDR
+	hex "Flash memory start address"
+	default 0x0
+	---help---
+		Start address of Flash memory
+
+if BUILD_PROTECTED
+
+config SRAM_START_ADDR
+	hex "RAM memory start address"
+	default 0x0
+	---help---
+		Start address of RAM memory to be used by the linker script. If multiple RAM are available,
+		then this config gives the start address of the RAM region for which the kernel and user
+		binaries need to be linked during the build process.
+
+config KSRAM_SIZE
+	int "Kernel RAM size in MB"
+	default 0
+	---help---
+		Size of kernel RAM region in KB.
+
+config USRAM_SIZE
+	int "User RAM size in MB"
+	default 0
+	---help---
+		Size of user RAM region in KB.
```

Comment:
Could you leave more detail for this config? What's the purpose of this, what is difference between heap and this, or something else.

Reply:
ok. Done